src/main/java/seedu/recruittrackpro/model/person/Comment.java

```diff
@@ -0,0 +1,59 @@
+package seedu.recruittrackpro.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.recruittrackpro.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's comment in the address book.
```

Comment:
Wanna clarify on this JavaDoc statement, to make it consistent should it be the following?
```suggestion
 * Represents a Person's comment in the address book.